sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Content/ContainerRegistryContentClient.cs

```diff
@@ -533,32 +533,32 @@ private static bool TryGetLength(Stream content, out long length)
         }
 
         /// <summary>
-        /// Downloads a manifest.
+        /// Gets a manifest.
         /// </summary>
-        /// <param name="tagOrDigest">The tag or digest of the manifest to download.</param>
-        /// <param name="mediaTypes">The set of media types to accept for the manifest being downloaded.</param>
+        /// <param name="tagOrDigest">The tag or digest of the manifest to get.</param>
+        /// <param name="mediaTypes">The set of media types to accept for the manifest.</param>
         /// <param name="cancellationToken">The cancellation token to use.</param>
         /// <returns>The download manifest result.</returns>
-        public virtual Response<DownloadManifestResult> DownloadManifest(string tagOrDigest, IEnumerable<ManifestMediaType> mediaTypes, CancellationToken cancellationToken = default)
+        public virtual Response<GetManifestResult> GetManifest(string tagOrDigest, IEnumerable<ManifestMediaType> mediaTypes, CancellationToken cancellationToken = default)
```

Comment:
out-of-scope of this Pr, but based on the feedback from the service team, we should remove this and do now allow requesting specific types

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/35083 to track.